src/app/app.component.ts

```diff
@@ -39,6 +42,14 @@ export class AppComponent implements OnInit, AfterViewInit {
           // update canonical url with query params as well
 
           this.updateCanonical(url.toLowerCase());
+          console.log('url', url);
+          if(url.includes('og/')) {
```

Comment:
can we do a more exact match? 
e.g. if I understand this correctly, `event?.urlAfterRedirects` should start with `/og/`